### PR TITLE
feat: [PL-25329]: Added Multitype input for notifications in user groups

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
##### Screenshots:

Screenshot 2022-06-14 at 2 39 58 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
dockerize

File: src/modules/20-rbac/components/NotificationList/NotificationList.tsx [hunk 1]
```diff
@@ -176,6 +178,32 @@ const ChannelRow: React.FC<ChannelRow> = ({
     }
   }
 
+  const renderInputField = (type: NotificationSettingConfigDTO['type']) => {
+    const { name, textPlaceholder } = getFieldDetails(type)
+    if (name === 'groupEmail') {
```
Comment (abhinavrastogi-harness) [inline src/modules/20-rbac/components/NotificationList/NotificationList.tsx:183]: we should use the `name` of an input field for changing behaviour.
let's pass a new prop/enum to change behaviour instead.

**Comment by avinashmadhwani02** [comment]
dockerize

**Comment by avinashmadhwani02** [comment]
trigger cypress

**Review by abhinavrastogi-harness — APPROVED** [review]

**Comment by avinashmadhwani02** [comment]
trigger cypress

**Review by mayankVermaHarness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**